multichain-api-gateway/config/base.toml

```diff
@@ -3,7 +3,7 @@ addr = "0.0.0.0:8044"
 
 [blockscout]
 base_url = "https://blockscout.com"
-concurrent_requests = 2
+concurrent_requests = 10
 instances = [
     ["eth", "mainnet"],
     ["xdai", "testnet"],
```

Comment:
I believe, it does not work, if you try to run your service with `MULTICHAIN_API_GATEWAY__CONFIG=config/base.toml cargo run`

Reply:
What maybe if you use not the array notation you use now, but just require to provide net and chain information in the way it is currently provided into url path (e.g. `eth/mainet`)? And then parse it with custom `#[serde(deserialize_with=custom_function)]` (https://serde.rs/field-attrs.html#deserialize_with) instead of `#[serde(with = "As::<Vec<DisplayFromStr>>")]` in BlockscoutSettings.instances part

Reply:
Let's though fix that in another PR maybe